File: MobiTaskClient/app/src/main/java/com/task/mobi/customViews/MyEditText.java
package com.task.mobi.customViews;

import android.content.Context;
import android.graphics.Typeface;
import android.util.AttributeSet;

public class MyEditText extends android.support.v7.widget.AppCompatEditText {
    Context context;
    int defStyleAttr;
    private AttributeSet attrs;

    public MyEditText(Context context) {
        super(context);
        Typeface face = Typeface.createFromAsset(context.getAssets(), "fonts/Roboto-Regular.ttf");
        this.setTypeface(face);
        this.context = context;
        init();
    }

    public MyEditText(Context context, AttributeSet attrs) {
        super(context, attrs);
        this.context = context;
        this.attrs = attrs;
        init();
    }

    public MyEditText(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        this.context = context;
        this.attrs = attrs;
        this.defStyleAttr = defStyleAttr;
        init();
    }

    @Override
    public void setTypeface(Typeface tf) {
        tf = Typeface.createFromAsset(getContext().getAssets(), "fonts/Roboto-Regular.ttf");
        super.setTypeface(tf);
    }

    @Override
    public void setTypeface(Typeface tf, int style) {
        tf = Typeface.createFromAsset(getContext().getAssets(), "fonts/Roboto-Regular.ttf");
        super.setTypeface(tf, style);
    }

    private void init() {
        Typeface font = Typeface.createFromAsset(getContext().getAssets(), "fonts/Roboto-Regular.ttf");
        this.setTypeface(font);
    }
}

File: MobiTaskAdmin/app/src/main/java/com/task/mobiadmin/customViews/MyEditText.kt
package com.task.mobiadmin.customViews

import android.content.Context
import android.graphics.Typeface
import android.util.AttributeSet

class MyEditText : android.support.v7.widget.AppCompatEditText {

    constructor(context: Context) : this(context, null)

    constructor(context: Context, attrs: AttributeSet?) : this(context, attrs, android.R.attr.editTextStyle)

    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {
        init()
    }

    override fun setTypeface(tf: Typeface) {
        super.setTypeface(getFont())
    }

    override fun setTypeface(tf: Typeface, style: Int) {
        super.setTypeface(getFont(), style)
    }

    private fun init() {
        this.typeface = getFont()
        includeFontPadding = false
    }

    private fun getFont() = Typeface.createFromAsset(context.assets, "fonts/Roboto-Regular.ttf")
}
